packages/now-go/go-helpers.ts

```diff
@@ -85,10 +86,17 @@ class GoWrapper {
     return this.execute(...args);
   }
 
-  build(src: string | string[], dest: string, ldsflags = '-s -w') {
+  build(src: string | string[], dest: string) {
     debug('Building optimized `go` binary %o -> %o', src, dest);
     const sources = Array.isArray(src) ? src : [src];
-    return this.execute('build', '-ldflags', ldsflags, '-o', dest, ...sources);
+
+    let flags = ['-ldflags', '-s -w'];
+    if (process.env.GO_BUILD_FLAGS) {
+      let flagsArr = stringArgv(process.env.GO_BUILD_FLAGS);
+      flags = [...flagsArr];
```

Comment:
I think you can just assign here instead of cloning:

```js
const flags = process.env.GO_BUILD_FLAGS?
  ? stringArgv(process.env.GO_BUILD_FLAGS)
  : ['-ldflags', '-s -w'];
```

Reply:
Updated 🙌